.stylelintrc

```diff
@@ -58,25 +58,20 @@
     "max-nesting-depth": 3,
     "media-feature-colon-space-after": "always",
     "media-feature-colon-space-before": "never",
-    "media-feature-no-missing-punctuation": true,
     "media-feature-range-operator-space-after": "always",
     "media-feature-range-operator-space-before": "always",
     "media-query-list-comma-newline-after": "always-multi-line",
     "media-query-list-comma-space-after": "always-single-line",
     "media-query-list-comma-space-before": "never",
-    "media-query-parentheses-space-inside": "never",
     "no-descending-specificity": true,
     "no-duplicate-selectors": true,
     "no-eol-whitespace": true,
     "no-invalid-double-slash-comments": true,
-    "no-missing-eof-newline": true,
     "no-unknown-animations": true,
     "number-leading-zero": "never",
     "number-max-precision": 2,
     "number-no-trailing-zeros": true,
-    "rule-non-nested-empty-line-before": ["always", {
-      "ignore": ["after-comment"]
-    }],
```

Comment:
This one changed to `rule-empty-line-before` and I think it's mandatory.

Reply:
Ok I will replace that.